### PR TITLE
add Faster rcnn c5

### PR DESCRIPTION
**Pull request by GT9505**
The C5 with mstrain and 1x schedule achieves 37.4 mAP, which is the same as model in Detectron2 (37.3 mAP).

### CODE REVIEW

File: mmdet/models/necks/channel_mapper.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+import torch.nn as nn
+from mmcv.cnn import ConvModule, xavier_init
+
+from ..builder import NECKS
+
+
+@NECKS.register_module()
+class ChannelMapper(nn.Module):
+    r"""Channel Mapper
```
Comment (ZwwWayne) [inline mmdet/models/necks/channel_mapper.py:9]: Channel Mapper to reduce channels of backbone features.

File: mmdet/models/necks/channel_mapper.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+
+    This is used to reduce channels of backbone features.
+
+    Args:
+        in_channels (List[int]): Number of input channels per scale.
+        out_channels (int): Number of output channels (used at each scale).
+        kernel_size (int): kernel_size for reducing channels (used at each
```
Comment (ZwwWayne) [inline mmdet/models/necks/channel_mapper.py:16]: 1. kernel_size (int, optional)
2. Defaults to 3.

File: mmdet/models/necks/channel_mapper.py [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+    def __init__(self,
+                 in_channels,
+                 out_channels,
+                 kernel_size=3,
+                 conv_cfg=None,
+                 norm_cfg=None,
+                 act_cfg=dict(type='ReLU')):
+        super(ChannelMapper, self).__init__()
+        assert isinstance(in_channels, list)
```
Comment (ZwwWayne) [inline mmdet/models/necks/channel_mapper.py:47]: Test this assertion in the unit test.

File: mmdet/models/necks/channel_mapper.py [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    This is used to reduce channels of backbone features.
+
+    Args:
+        in_channels (List[int]): Number of input channels per scale.
+        out_channels (int): Number of output channels (used at each scale).
+        kernel_size (int): kernel_size for reducing channels (used at each
+            scale).
+        conv_cfg (dict): Config dict for convolution layer. Default: None.
+        norm_cfg (dict): Config dict for normalization layer. Default: None.
+        act_cfg (dict): Config dict for activation layer in ConvModule.
```
Comment (ZwwWayne) [inline mmdet/models/necks/channel_mapper.py:20]: act_cfg (dict, optional)

**Review by ZwwWayne — APPROVED** [review]

File: mmdet/models/necks/channel_mapper.py [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+from mmcv.cnn import ConvModule, xavier_init
+
+from ..builder import NECKS
+
+
+@NECKS.register_module()
+class ChannelMapper(nn.Module):
+    r"""Channel Mapper to reduce channels of backbone features.
+
+    This is used to reduce channels of backbone features.
```
Comment (hellock) [inline mmdet/models/necks/channel_mapper.py:11]: not necessarily "reduce"

**Pull request merged**